recipes/qt/6.x.x/conanfile.py

```diff
@@ -198,6 +200,7 @@ def configure(self):
             del self.options.with_libpng
             del self.options.with_md4c
             self.options.rm_safe("with_x11")
+            self.options.rm_safe("with_egl")
```

Comment:
Should this option also be removed on macOS? I don't think EGL is available there.

Reply:
that's what line 169 does right ? (the change above the one you commented)

Reply:
My bad, I missed that. Thanks!